modules/channel-telegram/src/backend/client.ts

```diff
@@ -1,20 +1,30 @@
 import * as sdk from 'botpress/sdk'
 import _ from 'lodash'
-import path from 'path'
-import Telegraf, { Button, CallbackButton, ContextMessageUpdate, Markup } from 'telegraf'
-import Extra from 'telegraf/extra'
-
-import { Clients } from './typings'
-
-const outgoingTypes = ['text', 'typing', 'image', 'login_prompt', 'carousel']
+import Telegraf, { ContextMessageUpdate } from 'telegraf'
+import { TelegramCarouselRenderer } from '../renderers/carousel'
+import { TelegramChoicesRenderer } from '../renderers/choices'
+import { TelegramImageRenderer } from '../renderers/image'
+import { TelegramTextRenderer } from '../renderers/text'
+import { TelegramCommonSender } from '../senders/common'
+import { TelegramTypingSender } from '../senders/typing'
```

Comment:
Again, this is picky, but can you import all the renderers from a single rederers/index.ts file instead?

Reply:
Ok